frontends/mit-open/src/pages/HomePage/HomePage.tsx

```diff
@@ -1,25 +1,44 @@
 import React from "react"
 import { Container, styled } from "ol-components"
 import HeroSearch from "./HeroSearch"
-import FeaturedResourcesSection from "./FeaturedResourcesSection"
-import MediaSection from "./MediaSection"
 import BrowseTopicsSection from "./BrowseTopicsSection"
 import NewsEventsSection from "./NewsEventsSection"
+import ResourceCarousel from "@/page-components/ResourceCarousel/ResourceCarousel"
+import * as carousels from "./carousels"
 
 const FullWidthBackground = styled.div({
   background: "linear-gradient(0deg, #FFF 0%, #E7EBEE 100%);",
 })
 
+const FeaturedCoursesCarousel = styled(ResourceCarousel)(({ theme }) => ({
+  margin: "80px 0",
+  [theme.breakpoints.down("sm")]: {
+    marginTop: "0px",
+    marginBottom: "32px",
+  },
+}))
+const MediaCarousel = styled(ResourceCarousel)(({ theme }) => ({
+  margin: "80px 0",
+  [theme.breakpoints.down("sm")]: {
+    margin: "40px 0",
+  },
+}))
+
 const HomePage: React.FC = () => {
   return (
     <>
       <FullWidthBackground>
         <Container>
           <HeroSearch />
-          <FeaturedResourcesSection />
+          <FeaturedCoursesCarousel
```

Comment:
This is a bug in the resource card code and not your PR but maybe you can fix it here:

When I switch to a  view that has some resources without images I get a an error page. images! should be images? in the line below

https://github.com/mitodl/mit-open/blob/main/frontends/ol-components/src/components/LearningResourceCard/LearningResourceCard.tsx#L154